### PR TITLE
Gilles node syntax 2019 08

### PR DESCRIPTION
**Pull request by geomagilles**
This version includes the new syntax, but without present it - it should be 100% compatible with production. This PR is to make easier to add current features both syntaxes

### CODE REVIEW

**Comment by geomagilles** [comment]
@jalric could you check why a test is failing - it's not obvious for me

File: src/Code/yield/Client/Client.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const Alfred = require("./Alfred");
+const Run = require("./Run");
+const Select = require("./Select");
+const Schedule = require("./Schedule");
+const Connector = require("./Connector");
+const clientManager = require("./ClientManager");
+const objectify = require("../Services/Objectify");
+const { ExternalZenatonError } = require("../../../Errors");
+
+const Client = class Client {
```
Comment (jalric) [inline src/Code/yield/Client/Client.js:10]: `const Client = class Client`
This is redundant. You can just do
```javascript
class Client {
...
}
module.exports = Client;
```

File: src/Code/yield/Client/ClientManager.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const { ExternalZenatonError } = require("../../../Errors");
+
+let instance;
+
+const ClientManager = class ClientManager {
+  constructor() {
```
Comment (jalric) [inline src/Code/yield/Client/ClientManager.js:6]: Same as above.

File: src/Code/yield/Client/Connector.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+const uuidv4 = require("uuid/v4");
+const { ExternalZenatonError } = require("../../../Errors");
+
+const Connector = class Connector {
+  constructor(service, serviceId, processor) {
```
Comment (jalric) [inline src/Code/yield/Client/Connector.js:5]: Same as above.

File: src/Code/yield/Client/Run.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
+const { ExternalZenatonError } = require("../../../Errors");
+
+const MAX_ID_SIZE = 256;
+
+const Run = class Run {
+  constructor(processor) {
```
Comment (jalric) [inline src/Code/yield/Client/Run.js:6]: Same as above.

File: src/Code/yield/Client/Schedule.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
+const { ExternalZenatonError } = require("../../../Errors");
+
+const MAX_ID_SIZE = 256;
+
+const Schedule = class Schedule {
+  constructor(cron, processor) {
```
Comment (jalric) [inline src/Code/yield/Client/Schedule.js:6]: Same as above.

File: src/Code/yield/Client/Select.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
+const { ExternalZenatonError } = require("../../../Errors");
+
+const Select = class Select {
+  constructor(processor) {
```
Comment (jalric) [inline src/Code/yield/Client/Select.js:4]: Same as above.

File: src/Code/yield/Decider/Wait.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
+const moment = require("moment-timezone");
+const { ExternalZenatonError } = require("../../../Errors");
+
+const Wait = class Wait {
+  constructor(processor) {
```
Comment (jalric) [inline src/Code/yield/Decider/Wait.js:5]: Same as above.

File: src/Code/yield/Decider/WorkflowManager.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
+const versioner = require("../Services/Versioner");
+const { ExternalZenatonError } = require("../../../Errors");
+
+let instance;
+
+const WorkflowManager = class WorkflowManager {
+  constructor() {
```
Comment (jalric) [inline src/Code/yield/Decider/WorkflowManager.js:7]: Same as above.

File: src/Code/yield/Worker/TaskManager.js [hunk 9]
```diff
@@ -0,0 +1,53 @@
+const versioner = require("../Services/Versioner");
+const { ExternalZenatonError } = require("../../../Errors");
+
+let instance;
+
+const TaskManager = class {
```
Comment (jalric) [inline src/Code/yield/Worker/TaskManager.js:6]: Strange way to define a class. You can just do `class TaskManager` or I miss something.

**Pull request merged**